ansible/roles_ocp_workloads/ocp4_workload_lpe_3scale/files/install_3scale_tenant.yml

```diff
@@ -3,19 +3,19 @@
 
 # Name of 3scale API administrator of the tenant
 - set_fact:
```

Comment:
Technically I'd like a name on the task... but this is pretty self explanatory I guess.

Reply:
Done